src/main/java/org/primeframework/mvc/action/result/SaveRequestResult.java

```diff
@@ -138,6 +159,8 @@ public static class SaveRequestImpl implements SaveRequest {
 
     private final String uri;
 
+    private boolean allowPost;
```

Comment:
This is never set, so it is always `false`. You will need to add this to the constructor, and then update the tests. 

See `SaveRequestResultTest` - and add a case where we set this equal to `true` - similar to `saveRequestPOST` but for `allowPost = true`. 

Or maybe just keep the old version of `saveRequestPOST()` for when `allowPost = true`.

Reply:
updated